layouts/_default/board-of-directors-page.html

```diff
@@ -67,8 +67,49 @@ <h2 class="mb-5 text-5xl font-black">{{ . }}</h2>
   {{ $author := . }}
   <li
     id="{{ $author.slug }}"
-    class="row-subgrid row-span-3 grid auto-rows-min grid-cols-5 gap-x-4 gap-y-8 rounded bg-white p-6 shadow-md sm:max-md:row-span-2 lg:row-span-2"
+    class="row-subgrid relative  grid flex-[0_0_calc(50%_-_2rem)] auto-rows-min grid-cols-5 gap-x-4 gap-y-8 rounded bg-white p-6 pb-12 shadow-md"
+    x-data="{ isOpen: false }"
+    :style="!isOpen && { height: '640px' }"
+    :class="{'overflow-hidden': !isOpen}"
   >
+    <div
+      class="absolute inset-x-0 bottom-0 bg-gradient-to-t from-white to-transparent px-4 pt-12 pb-4"
+    >
+      <button
+        x-show="!isOpen"
+        :aria-expanded="''+isOpen"
+        class="group leading-4 text-navy hover:text-yellow"
+        @click="isOpen = true"
+        x-report-click
+        data-ga-category="toggle-bio"
+      >
+        <span aria-hidden="true">
+          <svg class="inline-block h-4 w-4 fill-current">
+            {{ partial "helper/svg.html" "plus" }}
+          </svg>
+        </span>
+        <span class="ml-2 align-bottom uppercase hover:underline">
+          Show more…
+        </span>
+      </button>
+      <button
+        x-show="isOpen"
+        :aria-expanded="''+isOpen"
```

Comment:
@carlmjohnson This button closes the bio to show less. What's the correct aria label here?

Reply:
You could add an ID to the "Show less…" span and then do `aria-label="{{ $showLessID }}"` here.

Reply:
Does it need an aria-label? It might just be okay without it, since that's the content anyway…

Reply:
These aria labels are removed